<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportQuizlet</class>
 <widget class="QDialog" name="ImportQuizlet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Von Quizlet importieren</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../ressources.qrc">
    <normaloff>:/icons/res/icons/language_FILL1_wght400_GRAD0_opsz24.svg</normaloff>:/icons/res/icons/language_FILL1_wght400_GRAD0_opsz24.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_3">
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Von Quizlet importieren</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Link zum Set:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lbl_sprache">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Sprache:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmb_sprache">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <item>
        <property name="text">
         <string>Englisch</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Französisch</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Latein</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Spanisch</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Italienisch</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="txt_link">
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="box_vertauschen">
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Spalte &quot;Begriff&quot; und &quot;Definition&quot; von Quizlet vertauschen</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="box_definition">
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>&quot;Definition&quot; von Quizlet in der Definitionsspalte des Vocitrainers speichern</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="styleSheet">
      <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Information: Die Quizlet-Webseite automatisierte Aufrufe blockt, öffnet der Vocitrainer beim Importvorgang den Firefox Browser um einen echten Aufruf zu simulieren. Ist dieser nicht installiert, kann der Import fehlschlagen.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="styleSheet">
      <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Information: Der Vocitrainer extrahiert das Set aus der Quizlet-Webseite. Wenn Quizlet Änderungen an seiner Webseite vornimmt, kann es sein, dass der Import fehlschlägt. Prüfe in diesem Fall, ob es für den Vocitrainer Updates gibt.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>229</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cmd_importieren">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Importieren</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cmd_abbrechen">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Abbrechen</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../ressources.qrc"/>
 </resources>
 <connections/>
</ui>
